xblockutils/base_test.py

```diff
@@ -89,7 +94,7 @@ def set_scenario_xml(xml):
         SCENARIOS.clear()
         add_xml_scenario("test", "Test Scenario", xml)
 
-    def go_to_view(self, view_name='student_view', student_id=None):
+    def go_to_view(self, view_name='student_view', student_id="student_1"):
```

Comment:
@smarnach I believe it should be fine. Although the URL is different, I'm not aware of any tests that depend on the URL. What's more, passing in no student_id makes workbench use `student_1` as a default, so this is just a more explicit version of the existing behaviour.